app/components/export-tool.js

```diff
@@ -82,44 +82,54 @@ export default Ember.Component.extend({
     _convertToJSON(dataArray) {
         return JSON.stringify(dataArray, null, 4);
     },
-    _convertToTSV(dataArray) {
+    _convertToCSV(dataArray) {
         // Flatten the dictionary keys for readable column headers
         let squashed = dataArray.map((item => squash(item)));
 
         var fields = Object.keys(squashed[0]);
-        var tsv = [fields.join('\t')];
+        var csv = [fields.join(',')];
 
         squashed.forEach((item) => {
             var line = [];
             fields.forEach(function(field) {
-                line.push(JSON.stringify(item[field]));
+                var value = JSON.stringify(item[field]).replace(/\\"/g, '""');
```

Comment:
There may be additional edge cases to consider. For example, what if the field contains a comma?

This document may be helpful for you (see section 2 in particular):
https://tools.ietf.org/html/rfc4180#section-2